streams/src/main/java/org/apache/kafka/streams/kstream/internals/KGroupedStreamImpl.java

```diff
@@ -489,7 +489,7 @@ public V apply(final K aggKey, final V aggOne, final V aggTwo) {
 
     }
 
-    @SuppressWarnings("deprecation")
+    @SuppressWarnings({"deprecation", "unchecked"})
```

Comment:
Ditto as before, I think the root cause is we should not enforce `StateStore S` template in `ProcessDetails`. And then here we could call `ProcessDetails.<K, T>builder()` to maintain typing enforcement.